tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+: {$KUBEVIRTNAMESPACE:="openshift-cnv"}
+_kubectl=$(which kubectl)
+_oc=$(which oc)
+
+[[ -z ${_kubectl} && -z ${_oc} ]] && echo "OC or KUBECTL is not installed" && exit 1
+
+[ -z "${KUBECONFIG}" ] && echo "KUBECONFIG is not set" && exit 1
+
+cat << EOF > test_config.json
+{
+ "storageClassLocal": "hostpath-provisioner",
+ "storageClassHostPath": "hostpath-provisioner",
+ "storageClassRhel": "hostpath-provisioner",
+ "storageClassWindows": "hostpath-provisioner",
+ "manageStorageClasses": false
+}
+EOF
+
+oc create namespace kubevirt
+rm -rf _out
+hack/dockerized hack/build-func-tests.sh
+make manifests
+rm -f _out/manifests/testing/cdi-*
+rm -f _out/manifests/testing/kubevirt-config.yaml
+sed -i "s/namespace: kubevirt/namespace: ${KUBEVIRTNAMESPACE}/g" _out/manifests/testing/*.yaml
+_out/tests/tests.test --cdi-namespace=${KUBEVIRTNAMESPACE} \
```

Comment:
We need a way to pass extra Ginkgo arguments. e.g. to select specific tests.

Reply:
I suggest to do this in a following PR because it requires testing.